templates/firefox-ios.xliff

```diff
@@ -1925,6 +1926,10 @@
         <source>This will open in another application</source>
         <note>Opening an external app</note>
       </trans-unit>
+      <trans-unit id="To use a custom Firefox Account and sync servers, specify the root Url of the Firefox Account site. This will download the configuration and setup this device to use the new service. After the new service has been set, you will need to create a new Firefox Account or login with an existing one.">
+        <source>To use a custom Firefox Account and sync servers, specify the root Url of the Firefox Account site. This will download the configuration and setup this device to use the new service. After the new service has been set, you will need to create a new Firefox Account or login with an existing one.</source>
```

Comment:
* "login" -> should be "log in" since it's used as a verb there
* "setup" -> same thing, should be "set up"